text/0000-tiered-browser-support.md

```diff
@@ -0,0 +1,289 @@
+such as [Windows Phone 8.1][win-phone], a device that is no longer supported. I
+made the decision that it wasn't worth it for the community to spend time
+fixing this issue, did I make the correct tradeoff for the community?
+
+[win-phone]: https://github.com/rust-lang/crates.io/issues/56
+
+Supporting all versions of all browsers with the same behavior is impossible
+with the small number of people who work on Rust's web content. Crates.io is
+not currently doing any cross-browser testing; there are some JavaScript tests
+of the UI that run in [PhantomJS][], a headless WebKit. Since we're not
+testing, we don't actually know what our current web support even is, except
+for when we get bug reports from users.
+
+[PhantomJS]: http://phantomjs.org/
+
+In order to fully test on all browsers to be sure of our support, we would
+either need to have all the devices, operating systems, browsers, and versions
+available and people with the time and inclination to do manual testing on all
+of these, or we would need to be running automated tests on something like
+[BrowserStack][]. BrowserStack does appear to have a free plan for open source
+projects, but it's unclear how many parallel tests the open source plan would
+give us, and we'd at least be spending time waiting for test results on the
+various stacks. [BrowserStack also doesn't support every platform][bs-support],
+Linux on the desktop being a notable section of our userbase missing from their
+platforms.
+
+[BrowserStack]: https://www.browserstack.com/pricing
+[bs-support]: https://www.browserstack.com/support
+
+# Detailed design
+[design]: #detailed-design
+
+## Rust web content
+
+Officially produced web content includes:
+
+- rust-lang.org
+- blog.rust-lang.org
+- play.rust-lang.org
+- crates.io
+- Rustdoc output
+- thanks.rust-lang.org
+
+Explicitly not included:
+
+- Content for people working on Rust itself, such as:
+  - [The Rust Forge][]
+  - [rusty-dash][]
+
+[The Rust Forge]: https://forge.rust-lang.org/
+[rusty-dash]: https://rusty-dash.com/
+
+Things that are not really under our control but are used for official or
+almost-official Rust web content:
+
+- GitHub
+- docs.rs
+- Discourse (used for [urlo][] and [irlo][])
+- [mdBook][] output (used for the books and other documentation)
+
+[urlo]: https://users.rust-lang.org/
+[irlo]: https://internals.rust-lang.org/
+[mdBook]: https://github.com/azerupi/mdBook/
+
+## Proposed browser support tiers
+
+Based on [actual usage metrics][] and with a goal of spending our time in an
+effective way, the browser support tiers would be defined as:
+
+[actual usage metrics]: #google-analytics-browser-usage-stats
+
+### Tier 1
+
+Goal: Ensure functionality of our web content for 80% of users.
+
+Browsers:
+
+- Chrome current stable and one previous release
+- Firefox current stable and the latest ESR
+- Safari current stable, one previous minor release, and the current version on
+  iOS
+- UC Browser current stable
+
```

Comment:
I think we should include Edge on this list - MS has big market share and we want to expand on Windows. Currently this feels like more of treating windows users second class.